tests/test_virtualenv.py

```diff
@@ -332,3 +334,40 @@ def test_relative_symlink(tmpdir):
     shutil.rmtree(workdir)
 
     assert os.path.exists(lib64)
+
+
+def test_missing_certifi_pem(tmpdir):
```

Comment:
use the new ``tmp_path`` please instead of ``tmpdir``. 👍

Reply:
the test above uses tmpdir. what is tmp_path?

Reply:
we'll migrate the old tests sometime in the future, but no point in not already use it for new tests - https://docs.pytest.org/en/latest/tmpdir.html?highlight=tmp_path#the-tmp-path-fixture

Reply:
oh, pathlib! may i use pathlib also instead of the os.path functions?

Reply:
Done.

Reply:
yeah 👍